user-interface/src/case-assignment/AssignAttorneyCasesRow.tsx

```diff
@@ -0,0 +1,136 @@
+import { useEffect, useState } from 'react';
+import { TableRow, TableRowData, TableRowProps } from '@/lib/components/uswds/Table';
+import { ToggleModalButton } from '@/lib/components/uswds/modal/ToggleModalButton';
+import { CaseNumber } from '@/lib/components/CaseNumber';
+import { formatDate } from '@/lib/utils/datetime';
+import Button, { UswdsButtonStyle } from '@/lib/components/uswds/Button';
+import { CaseWithAssignments } from './CaseAssignmentScreen.types';
+import { AssignAttorneyModalRef } from './AssignAttorneyModal';
+import { useApi2 } from '@/lib/hooks/UseApi2';
+import { LoadingSpinner } from '@/lib/components/LoadingSpinner';
+
+export type AssignAttorneyCasesRowProps = TableRowProps & {
+  bCase: CaseWithAssignments;
+  idx: number;
+  modalId: string;
+  modalRef: React.RefObject<AssignAttorneyModalRef>;
+};
+
+export function AssignAttorneyCasesRow(props: AssignAttorneyCasesRowProps) {
+  const { bCase, idx, modalId, modalRef, ...otherProps } = props;
+
+  const [internalCase, setBCase] = useState<CaseWithAssignments>(bCase);
+  const [inTableTransferMode] = useState<boolean>(false);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+
+  const api = useApi2();
+
+  useEffect(() => {
+    api
+      .getCaseAssignments(internalCase.caseId)
+      .then((response) =>
+        setBCase({
+          ...internalCase,
+          assignments: response.data.map((assignment) => assignment.name),
+        }),
+      )
+      .catch((_reason) => {})
+      .finally(() => {
+        setIsLoading(false);
+      });
+  }, []);
+
+  return (
+    <TableRow className={inTableTransferMode ? 'in-table-transfer-mode' : ''} {...otherProps}>
```

Comment:
Remove in table transfer mode?? It isn't working, and we are OK with that. Tech debt.